packages/shorebird_cli/lib/src/shorebird_android_artifacts.dart

```diff
@@ -0,0 +1,110 @@
+import 'package:path/path.dart' as p;
+import 'package:scoped/scoped.dart';
+import 'package:shorebird_cli/src/command.dart';
+import 'package:shorebird_cli/src/third_party/flutter_tools/lib/flutter_tools.dart';
+
+/// Throw when multiple artifacts are found in the build directory.
```

Comment:
Will that happen when using Android Studio and building different flavors/configs between multiple runs?  I wonder if older .aab's from other configs being in the build dir can cause these failures to happen?

Reply:
I should never happen as far I can tell, this was more of a "just to be sure" checking that Bryan suggested while we were discussing the solution.

Reply:
Not yet. Was focusing on our PR first, will open now that it seems that I addressed all/most of the comments

Reply:
Filled: https://github.com/flutter/flutter/issues/147261